pyomo/common/misc.py

```diff
@@ -0,0 +1,284 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and 
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain 
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
```

Comment:
Same comment from @jsiirola:

> 
> 
> Because this code was originally in PyUtilib, should we include part of the original PyUtilib copyright statement? Maybe something like:
> 
> ```python
> #  ___________________________________________________________________________ 
> #  This module was originally developed as part of the PyUtilib project
> #  Copyright (c) 2008 Sandia Corporation.
> #  This software is distributed under the BSD License.
> #  Under the terms of Contract DE-AC04-94AL85000 with Sandia Corporation,
> #  the U.S. Government retains certain rights in this software.
> #  ___________________________________________________________________________ 
> ```